src/pages/Calculations.svelte

```diff
@@ -1,48 +1,56 @@
 <Main>
-	{#each $calculations as calculation (calculation.id)}
-		<Tile centered={false}>
-			<svelte:fragment slot="title">
-				<h5 class="mt-2">{@html calculation.name}</h5>
-			</svelte:fragment>
-			<svelte:fragment slot="subtitle">
-				<small class="text-gray">
-					{showTimestamp(calculation)}
-				</small>
-			</svelte:fragment>
-			<svelte:fragment slot="action">
-				<IconButton
-					icon="cross"
-					color="error"
-					on:click={() =>
-						withConfirm(
-							delCalculation,
-							calculation.id,
-							'Are you sure?',
-							false
-						)?.(calculation.id)}
-				/>
-			</svelte:fragment>
-		</Tile>
-	{:else}
-		<div class="text-center distant_msg">No calculations</div>
-	{/each}
+	<div bind:clientWidth={width}>
+		{#await $calculationsAsync}
+			{#each { length: 4 } as _}
+				<Loaders.Tile count={1} w={width} h={74} height={74} {width} />
+			{/each}
+		{:then calculations}
+			{#each calculations as calculation (calculation.id)}
+				<Tile centered={false}>
+					<svelte:fragment slot="title">
+						<h5 class="mt-2">{@html calculation.name}</h5>
+					</svelte:fragment>
+					<svelte:fragment slot="subtitle">
+						<small class="text-gray">
+							{showTimestamp(calculation)}
+						</small>
+					</svelte:fragment>
+					<Meter value={calculation.status} min={0} max={100} low={25} high={50} />
```

Comment:
No needed `min,max,low,high` - they have this values by defaults.
But `striped, animated` needed.

```svelte
<Meter
   value={calculation.status}
   striped={calculation.status < 100}
   animated={calculation.status < 100}
/>
```